spec/models/queue_item_spec.rb

```diff
@@ -37,6 +38,34 @@
       end
     end
 
+    describe "#rating=" do
+      it "changes the rating of the review if the review is present" do
+        a_new_hope = Fabricate(:video)
+        luke = Fabricate(:user)
+        review = Fabricate(:review, user: luke, video: a_new_hope, rating: 2)
+        queue_item = Fabricate(:queue_item, user: luke, video: a_new_hope)
+        queue_item.rating = 4
+        expect(Review.first.rating).to eq(4)
+      end
+
+      it "clears the rating of the review if the review is present" do
+        a_new_hope = Fabricate(:video)
+        luke = Fabricate(:user)
+        review = Fabricate(:review, user: luke, video: a_new_hope, rating: 2)
+        queue_item = Fabricate(:queue_item, user: luke, video: a_new_hope)
+        queue_item.rating = nil
+        expect(Review.first.rating).to be_nil
+      end
+
+      it "creates a review with the rating if the review is not present" do
+        a_new_hope = Fabricate(:video)
+        luke = Fabricate(:user)
+        queue_item = Fabricate(:queue_item, user: luke, video: a_new_hope)
+        queue_item.rating = 3
+        expect(Review.first.rating).to eq(3)
```

Comment:
This spec doesn't really do what is described. I would change the description to something like `it "assigns the correct rating to the new review"`, then create another spec with the same setup and description as this one, and use this expectation:

```ruby
expect { queue_item.rating = 2 }
  .to change { Review.count }.from(0).to(1)
```

Alternatively, in this one case I would be okay with replacing line 64 with the line above and having two expectations in this spec.